### PR TITLE
test: Run karma tests with full CSS

### PR DESCRIPTION
**Pull request by aarongranick-okta**
### Description
- Adds `grunt css` command to build only CSS 
- remove 404s to "foo.com"
- fix case sensitive error "1x1.GIF"
### Screenshots

https://drive.google.com/open?id=1LEaASft_aQAoRH8bFAn3OEUQ9ed4flu-

### Reviewers
@haishengwu-okta @nikhilvenkatraman-okta

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -316,26 +316,41 @@ module.exports = function (grunt) {
...
       postBuildTasks.push('copy:target-to-dist');
     } else {
-      buildTasks.push('postcss:build', 'exec:build-dev');
+      buildTasks.push('exec:build-dev');
     }
     grunt.task.run([
       'exec:clean',
       'exec:retirejs',
       'exec:generate-config',
-      'copy:app-to-target',
```
Comment (haishengwu-okta) [inline Gruntfile.js]: iirc `generate-jsonp` depends on `copy:app-to-target`. Did you double check if your changes break anything?

Comment (aarongranick-okta) [reply Gruntfile.js]: thanks I'll check this.

Comment (aarongranick-okta) [reply Gruntfile.js]: @haishengwu-okta I have fixed this and also added tests to verify that the jsonp bundles exist, can be loaded and applied.

**Comment by aarongranick-okta** [comment]
Tests are passing on Bacon also: https://bacon.aue1e.internal/#!/commits/okta-signin-widget/branch/bacon-topic-ag

**Pull request merged**